### PR TITLE
created dropdown menu

### PR DESCRIPTION
**Pull request by hexflores87**

### CODE REVIEW

File: app/javascript/components/components/Header.js [hunk 1]
```diff
@@ -1,25 +1,62 @@
...
+//         {logged_in && <a href={sign_out_route}>Sign Out</a>}
+//         {!logged_in && <a href={sign_in_route}>Sign In</a>}
+//         <br></br>
+//         {this.props.logged_in && (
+//           <NavLink to="/userrecipes"> Go to your Recipes Page!</NavLink>
+//         )}
+//       </header>
+//     );
+//   }
+// }
```
Comment (minachuong) [inline app/javascript/components/components/Header.js]: if this portion of code is not being used anymore, it'd be helpful to keep the file clean by removing it

File: app/javascript/components/components/Header.js [hunk 2]
```diff
@@ -1,25 +1,62 @@
...
-            </header>
-        
-        )
-    }
-}
+  const toggleNavbar = () => setCollapsed(!collapsed);
 
-export default Header
+  //class Header extends Component {
+  // render() {
```
Comment (minachuong) [inline app/javascript/components/components/Header.js]: if this portion of code is not being used anymore, it'd be helpful to keep the file clean by removing it

File: app/javascript/components/components/Header.js [hunk 3]
```diff
@@ -1,25 +1,62 @@
...
+        </NavbarBrand>
+        <NavbarToggler onClick={toggleNavbar} className="hamburger-menu" />
+        <Collapse isOpen={!collapsed} navbar>
+          <Nav navbar>
+            <NavItem>
+              {logged_in && <a href={sign_out_route}>Sign Out</a>}
+              {!logged_in && <a href={sign_in_route}>Sign In</a>}
+            </NavItem>
+            <NavItem>
+              <NavLink to="/components/pages/Home">Home</NavLink>
```
Comment (minachuong) [inline app/javascript/components/components/Header.js]: I believe the path for the `to` prop should be using routes declared in `App.js`

File: app/javascript/components/components/Header.js [hunk 4]
```diff
@@ -1,25 +1,62 @@
...
+            </NavItem>
+            <NavItem>
+              <NavLink to="/components/pages/RecipeIndex">
```
Comment (minachuong) [inline app/javascript/components/components/Header.js]: I believe the path for the `to` prop should be using routes declared in `App.js`

File: app/javascript/components/components/Header.js (hunk 1 above)
Comment (JJSanch) [reply app/javascript/components/components/Header.js]: Got it

File: app/javascript/components/components/Header.js (hunk 2 above)
Comment (JJSanch) [reply app/javascript/components/components/Header.js]: Got it

File: app/javascript/components/components/Header.js (hunk 4 above)
Comment (JJSanch) [reply app/javascript/components/components/Header.js]: Got it

**Comment by hexflores87** [comment]
Changes made on the header.

**Review by minachuong — APPROVED** [review]

**Pull request merged**